### DISCUSSION TITLE
Hi! needed for friend

### DISCUSSION THREAD

**Discussion by end-less7** [godotengine/godot-proposals#7034] [General Discussions]
I am learning visual scripting from unity. I want some friends to make games. I feel along to make games and if I got any problems no-one is their to solve and talk to me. If you have age of 13 to 16 than you can join me. I only use visual scripting in unity. If you want to join me now than hangout to my my email that is ambareeshsahu18510@gmail.com. Do it now

**Comment by Calinou** [comment]
This is not a platform for support questions. Also, this is a Godot platform, not a Unity platform.
